src/swapanaW/AllArrayIndex.java

```diff
@@ -0,0 +1,40 @@
+package swapanaW;
+
+import java.util.Scanner;
+
+public class AllArrayIndex {
+
+	public static void main(String[] args) {
+
+		System.out.println("Please enter the first array values : ");
+		Scanner sc = new Scanner(System.in);
+		int[] array1 = new int[6];
+
+		for (int i = 0; i < array1.length; i++) {
+			array1[i] = sc.nextInt();
+		}
+		
+		System.out.println("Please enter the second array values : ");
+		int[] number = new int[6];
+
+		for (int i = 0; i < number.length; i++) {
+			number[i] = sc.nextInt();
+		}
+		
+		AllArrayIndex arrayIndex = new AllArrayIndex();
+		arrayIndex.indexarrays(array1, number);
+	}
+
+	void indexarrays(int[] array1, int[] number) {
+		int i = 0;
+		int j = 0;
+		while (i < array1.length) {
```

Comment:
Hi swapna 
Here we can check with simple for loop with n complexity 
Why we check condition in two while loop. Its complexity is n*n 
Which not applicable. Logic is correct to check indexing.